contrib/upload-kktix-ticket-csv-to-bigquery.py

```diff
@@ -322,5 +322,63 @@ def test_hash_email(self):
         )
 
 
+class Test2019Ticket(unittest.TestCase):
```

Comment:
Are we using pytest or not? If we are, this does not need to use `unittest.TestCase`. If we are not, tests should use `assertEqual()` etc. functions, not bare `assert`.

Reply:
Good catch! Should be assertEqual +1. It's the unittest of standard lib.

Reply:
Updated and re-based this pull request. Please refer to this commit https://github.com/pycontw/PyCon-ETL/pull/34/commits/2bf312b26a878cddc68e4e0ec05957ba4deefb0c implemented by following your suggestion.